vochain/state.go

```diff
@@ -561,6 +591,36 @@ func (v *State) incrementTreasurerNonce() error {
 	return v.Tx.DeepSet([]byte(TreasurerKey), tBytes, ExtraCfg)
 }
 
+func (v *State) SetTxCostByTxType(txType models.TxType, cost uint64) error {
```

Comment:
Given that this function calls internally `SetTxCost()` can we have `SetTxCost()` unexported ?
Is there any other way to set a tx cost that does not require to have a `TxType` ? Asking this because I cannot clearly see the fact that there are two ways of setting the tx cost. I there should be just one way to set this and is using the `txType`.

Reply:
Same for the get methods.

Reply:
OK, this will mean that everything will have to be translated to the models.TxType int32 when one wants to specify what type of cost one wants to update/get.

Reply:
Err, perhaps I should re-clarify.
As you may remember in our zeal we made `TransactionCosts{}` translate itself into a key-value map where the keys were "c_*". This is useful when initializing the State from genesis. That's what `SetTxCost(key string)` is for.

However, for the rest of the programs' lifetime, it will receive requests by `models.TxType`. So, maybe we should make `TransactionCosts{}.AsMap()` also use `models.TxType`, even though using "c_*" as keys is more... intuitive.